static/templates/language-learning.html

```diff
@@ -110,95 +110,95 @@ <h2 class="amarante font-lg">Full Lessons</h2>
             <a class="audio-file" href="/static/lessons/Audio-Lesson-1.mp3" download>Audio File</a>
           </div>
         </div>
         <div class="lesson">
           <h3 class="font-sm amarante">Intro Lesson 2</h3>
           <p class="font-xsm fira-sans">Learn to ask and answer questions about animals</p>
-          <div class="font-xsm">
+          <div class="font-xsm roboto">
             <a class="btn" href="/static/lessons/Binder-Eval-Lesson-2.pdf" download>PDF</a>
             <a class="audio-file" href="/static/lessons/Audio-Lesson-2.mp3" download>Audio File</a>
           </div>
         </div>
         <div class="lesson">
           <h3 class="font-sm amarante">Intro Lesson 3</h3>
           <p class="font-xsm fira-sans">Learn to construct more complicated questions about animals</p>
-          <div class="font-xsm">
+          <div class="font-xsm roboto">
             <a class="btn" href="/static/lessons/Binder-Eval-Lesson-3.pdf" download>PDF</a>
-            <a class="audio-file" href="lessons/Audio-Lesson-3.mp3" download>Audio File</a>
+            <a class="audio-file" href="static/lessons/Audio-Lesson-3.mp3" download>Audio File</a>
           </div>
         </div>
         <div class="lesson">
           <h3 class="font-sm amarante">Intro Lesson 4</h3>
           <p class="font-xsm fira-sans">Learn to ask answer questions about people</p>
-          <div class="font-xsm">
+          <div class="font-xsm roboto">
             <a class="btn" href="/static/lessons/Binder-Eval-Lesson-4.pdf" download>PDF</a>
             <a class="audio-file" href="/static/lessons/Audio-Lesson-4.mp3" download>Audio File</a>
           </div>
         </div>
         <div class="lesson">
           <h3 class="font-sm amarante">Intro Lesson 5</h3>
           <p class="font-xsm fira-sans">Learn to construct more complicated questions about people</p>
-          <div class="font-xsm">
+          <div class="font-xsm roboto">
             <a class="btn" href="/static/lessons/Binder-Eval-Lesson-5.pdf" download>PDF</a>
             <a class="audio-file" href="/static/lessons/Audio-Lesson-5.mp3" download>Audio File</a>
           </div>
         </div>
         <div class="lesson">
           <h3 class="font-sm amarante">Prayer</h3>
           <p class="font-xsm fira-sans">Learn a short prayer</p>
-          <div class="font-xsm">
+          <div class="font-xsm roboto">
             <a class="btn" href="/static/lessons/Prayer.pdf" download>PDF</a>
-            <a class="audio-file" href="/static/lessons/Audio-Prayer-Lesson.mp3" download>Audio File</a>
+            <a class="audio-file" href="/static/lessons/Audio-Lesson-Thankful-Prayer.mp3" download>Audio File</a>
           </div>
         </div>
         <div class="lesson">
           <h3 class="font-sm amarante">Bedroom</h3>
           <p class="font-xsm fira-sans">Learn about greetings - this is the lesson displayed above</p>
```

Comment:
I suspect this is not the right description for the "Bedroom" lesson. Can you check Figma and/or ask Lisa about this one?

Reply:
@cwinters8 I need to ask Lisa about this file